src/xtopcom/xbasic/src/xstring.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2017-2021 Telos Foundation & contributors
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "xbasic/xstring.h"
```

Comment:
We already have xstring_utl module, so why we add this part? Is it because the difference between xbase and xbasic?

Reply:
for writing unified code (template) in contract runtime.